shortfin/python/shortfin_apps/utils.py

```diff
@@ -166,14 +166,43 @@ class StrobeMessage(sf.Message):
     ...
 
 
-class ServiceBase:
+class GenerateService:
     """Base class for shortfin service implementations."""
 
     def __init__(self, sysman: SystemManager):
         """Initialize base service attributes."""
         self.sysman = sysman
         self.inference_parameters: dict[str, list[sf.BaseProgramParameters]] = {}
         self.inference_modules: dict[str, list[sf.ProgramModule]] = {}
+        self.name = None
+        self.model_params = None
```

Comment:
This should be an `__init__` argument. Generally, the base class should be responsible for initialization of its members.